### PR TITLE
Bounty upload/1 npm npm programmatic

### PR DESCRIPTION
**Pull request by benharvie**
If the Sheriffs would kindly validate this vulnerability before we merge it in as a bounty.

Much appreciated,
Ben ❤️

### CODE REVIEW

File: bounties/npm/npm-programmatic/1/vulnerability.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		"PR": "L",
+		"UI": "N",
+		"S": "U",
+		"C": "L",
+		"I": "L",
+		"A": "N",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "3.6"
```
Comment (Mik317) [inline bounties/npm/npm-programmatic/1/vulnerability.json:35]: Higher the `score` since it's a `arbitrary command execution`

**Review by Mik317 — APPROVED** [review]
Issue `verified` :smile:

I would suggest only to `higher` the `severity` and `CVSS score` since it's an `arbitrary command execution` :smile:

Cheers,
Mik

File: bounties/npm/npm-programmatic/1/vulnerability.json (hunk 1 above)
Comment (benharvie) [reply bounties/npm/npm-programmatic/1/vulnerability.json:35]: Thanks for pointing this out @Mik317, do you have a score in mind that would suit this vulnerability better?

Comment (Mik317) [reply bounties/npm/npm-programmatic/1/vulnerability.json:35]: Hey @benharvie :smile:, the similar issues on the platform have been considered around `7.5` and `8.0` basing on the possibility `affected method/function` could be used in a real-scenario.

Basing on this I would say `7.5` is ok :smile:

Cheers,
Mik

Comment (benharvie) [reply bounties/npm/npm-programmatic/1/vulnerability.json]: Updated as recommended, cheers! ❤️

**Pull request merged**